app/views/course/forum/posts/edit.html.slim

```diff
@@ -0,0 +1,3 @@
+= page_header
+
+= render partial: 'form', locals: { url: course_forum_topic_post_path(current_course, @forum, @topic, @post) }
```

Comment:
hmm... this form is used for both edit and new. So I need to pass different urls as locals...